Framework/Algorithms/inc/MantidAlgorithms/CalculateDynamicRange.h

```diff
@@ -0,0 +1,50 @@
+#ifndef MANTID_ALGORITHMS_CALCULATEDYNAMICRANGE_H_
+#define MANTID_ALGORITHMS_CALCULATEDYNAMICRANGE_H_
+
+#include "MantidAPI/Algorithm.h"
+#include "MantidAlgorithms/DllConfig.h"
+
+namespace Mantid {
+namespace Algorithms {
+
+/** CalculateDynamicRange
```

Comment:
New-style copyright notice? A helper Python script is available.

This comment may apply to other `.h` and `.cpp` files in this PR as well.